### ISSUE TITLE
User is not directed back to Product Opener after verifying their email address

### ISSUE THREAD

**Issue by john-gom** [openfoodfacts/openfoodfacts-auth#103]
After clicking the link the user is directed to their account page.

This is not the case for the "forget password" option where the email link redirects back to Product Opener after the user has changed their password.

**Comment by hangy** [comment]
Which entrypoint is causing the issue? The "sign in" button works fine for me: ![Screen capture of the sign-out process to ProductOpener using Keycloak](https://github.com/user-attachments/assets/70e16374-5664-49e4-87a3-f0d38785218c)

**Comment by john-gom** [comment]
OK, I was able to reproduce it. The basic problem is that we can't pass the locale into the account screen - it has to use the locale stored against the user. However, if the user doesn't change the locale in Keycloak then no value is stored, so the scenario to re-create the problem is:

1. Launch the Keycloak login process passing a locale in ui_locales=... (e.g. xx)
2. Register a new user and verify the email
3. Launch the account page from the app - the browser's default locale will be used

Fixed now with a test.
